packages/dom/src/css.ts

```diff
@@ -20,10 +20,10 @@
  * under the License.
  */
 
-import optimalSelect from 'optimal-select';
+import { finder } from '@medv/finder';
 import type { CssSelector, Matcher } from '@apache-annotator/selector';
-import { ownerDocument } from './owner-document';
-import { toRange } from './to-range';
+import { ownerDocument } from './owner-document.js';
+import { toRange } from './to-range.js';
```

Comment:
I like the explicitness of adding extensions, but it seems to break now when running `yarn run web:build` or `web:server`, with errors like:

> ERROR in ../packages/dom/src/index.ts
> Module not found: Error: Can't resolve './css.js' in '/vagrant/annotator/incubator-annotator/packages/dom/src'

Renaming all the `.js` imports to `.ts` seems to fix this for the `web:…` scripts; my editor (VSCodium) now does not understand the imports anymore though, not sure why. Also the linter now gives errors, not sure that is a step forward or back. Ideas?

Reply:
This issue should be resolved now.